src/style.css

```diff
@@ -1 +1,76 @@
-/* Styles go here */
+
+* {
+  margin: 0;
+}
+
+.moyo-container {
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.moyo-logo1 {
+  padding: 10px 0;
+}
+
+.main-header {
+  max-width: 1200px;
```

Comment:
redundant:
![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/3be4368b-4656-4f4a-bb3a-f57c496e89e0)

Reply:
still not fixed